web-components/src/[sandbox]/examples/datepicker.ts

```diff
@@ -16,6 +17,16 @@ const maxDate = now()
 
 export const datePickerTemplate = html`
   <h2 class="sandbox-header">md-datepicker</h2>
+  <h3>row calendar</h3>
+  <md-datepicker-calendar
```

Comment:
this allows to display the calendar outside of the popup which is way more practical